migrations/20240929040050-create-modificationtime-index-on-files.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const tableName = 'files';
+const indexName = 'files_modification_time_index';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
```

Comment:
I need these indexes to be created CONCURRENTLY @jzunigax2. Thus, I can run them in the background without locking the `folders` table. It's less clean but far more effective.

Reply:
Ref: https://www.postgresql.org/docs/current/sql-createindex.html